app/src/main/java/com/honeybeard/androidfundamentals/activities/movie_details/MovieDetailsFragment.kt

```diff
@@ -0,0 +1,51 @@
+package com.honeybeard.androidfundamentals.activities.movie_details
+
+import android.os.Bundle
+import androidx.fragment.app.Fragment
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import com.honeybeard.androidfundamentals.R
+
+// TODO: Rename parameter arguments, choose names that match
```

Comment:
please remove unused fields/methods/constants
this is decrease code readability